piksi_tools/console/system_monitor_view.py

```diff
@@ -32,10 +32,18 @@ def ip_bytes_to_string(ip_bytes):
 
 class SimpleAdapter(TabularAdapter):
     columns = [('Thread Name', 0), ('CPU %', 1), ('Stack Free', 2)]
+    can_edit = Bool(False)
+
+    def get_drag(self, object, trait, row):
+        return None  # disables drag
 
 
 class SimpleCSACAdapter(TabularAdapter):
     columns = [('Metric Name', 0), ('Value', 1)]
+    can_edit = Bool(False)
```

Comment:
this looks like shotgun surgery, can class level variables be inherited? maybe a superclass that all of these SimpleAdapters can inherit would reduce duplicated logic here.

Reply:
Agreed, it was a bit rushed. Refactored.